src/app-layout/visual-refresh-toolbar/skeleton/__tests__/skeleton-parts.test.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import {
+  NotificationsSkeleton,
+  ToolbarSkeleton,
+} from '../../../../../lib/components/app-layout/visual-refresh-toolbar/skeleton/skeleton-parts';
+
+describe('ToolbarSkeleton', () => {
+  test('renders with discoveredBreadcrumbs', () => {
+    const { container } = render(
+      <ToolbarSkeleton
+        appLayoutInternals={
+          {
+            breadcrumbs: null,
+            discoveredBreadcrumbs: { items: [] },
```

Comment:
If we need to cast `{}`, can we do it against the actual interface that the component asks for (`AppLayoutInternals`) instead of `any`?